docs/canvas/canvas-elements.asciidoc

```diff
@@ -0,0 +1,259 @@
+
+. Click *Timelion*.
+
+. Enter a *Query* using the Lucene query string syntax. 
++
+For for more information about the Lucene query string syntax, refer to <<lucene-query,Lucene Query Syntax>>.
+
+. Enter the *Interval*, then *Preview* the query.
+
+. If your query looks correct, *Save* it. 
+
+[float]
+[[configure-display-options]]
+=== Choose the display options
+
+Choose how you want the elements to appear on your workpad.
+
+[float]
+[[process-and-display]]
+==== Specify how to process and display the data
+
+When you connect your element to a data source, the element often appears as a warning. To remove the error, specify the process and display options.
+
+. Click *Display*
+
+. Change the data points to display in your element. 
+
+. In the *Label* field, enter a name for the data.
+
+[float]
+[[element-display-container]]
+==== Change the appearance of the element container
+
+Further define the appearance of the element container and border.
+
+. Next to *Element style*, click *+*, then select *Container style*.
+
+. Expand *Container style*.
+
+. Change the *Appearance* and *Border* options.
+ 
+[float]
+[[apply-element-styles]]
+==== Apply a set of styles
+
+To make your element look exactly the way you want, apply CSS overrides.
+
+. Next to *Element style*, click *+*, then select *CSS*.
+
+. Enter the *CSS*. For example, to center the Markdown element, enter:
++
+[source,js]
+--------------------------------------------------
+.canvasRenderE1 h1 {
+text.align: center;
+}
+--------------------------------------------------
+
+. Click *Apply stylesheet*.
+
+[float]
+[[configure-auto-refresh-interval]]
+==== Change the data auto-refresh interval
+
+Increase or decrease how often your data refreshes on your workpad.
+
+. In the top left corner, click the *Control settings* icon.
+
+. Under *Change auto-refresh interval*, select the interval you want to use.
++
+[role="screenshot"]
+image::images/canvas-refresh-interval.png[Element data refresh interval]
+
+TIP: To manually refresh the data, click the *Refresh data* icon. 
+
+[float]
+[[organize-element]]
+=== Organize the elements on your workpad
+
+Choose where you want the elements to appear on your workpad.
+
+[float]
+[[move-canvas-elements]]
+==== Move elements
+
+Move the element to a preferred location on your workpad. As you move the element, notice the alignment lines that appear to help you place the element exactly where you want it.
+
+* Click and drag the element to your preferred location.
+
+* To move the element by 1 pixel, select the element, then use your arrow keys.
+
+[float]
+[[resize-canvas-elements]]
+==== Resize elements
+
+Make your elements bigger or smaller than the default size.
+
+. Select the element.
+
+. Click and drag the resizing icon to the size you want.
+
+[float]
+[[align-canvas-elements]]
+==== Align elements
+
+Align two or more elements on your workpad.
+
+. Press Shift, then select the elements you want to align.
```

Comment:
We should say `Hold Shift` instead of press.

Reply:
Should Shift be in a code block since it's a keyboard key?

Reply:
Since they need to press before they hold, I changed all of these to `press and hold`. 

When we refer to keyboard keys, we always say Shift instead of `Shift`.